src/components/labwareScanPanel/columns.tsx

```diff
@@ -22,6 +22,14 @@ const color: ColumnFactory<Map<number, any>> = (meta) => {
   };
 };
 
+function joinUnique(array: string[]) {
+  return Array.from(new Set<string>(array)).join(", ");
+}
+
+function valueFromSamples(labware: LabwareLayoutFragment, sampleFunction: (sample: any) => (string)) {
```

Comment:
Rather than using `any`, you could use TypeScript's [indexed access types](https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html) for the `sample` argument here. (`LabwareLayoutFragment["slots"][number]["samples"][number]` I think)